shared/utils/combine-ovals.py

```diff
@@ -271,6 +272,13 @@ def check_is_loaded(loaded_dict, filename, version):
     return False
 
 
+def get_oval_version_from_oval(xml_content):
+    oval_version = "".join(re.findall("oval_version=\"\d+\.\d+\"", xml_content))
+
```

Comment:
I would prefer to get the OVAL version by getting the XML element and attribute using ElementTree, because it's an XML document. Why is regular expression used?

Reply:
Yeah, it would be nice to use ElementTree for OVAL definitions, but many of the checks are "malformed" from ElementTree point of view. These definitions have undefined namespace prefixes, like "ind" and "linux", and trigger unbounded prefix error.

Reply:
@jan-cerny what @yuumasato pointed out is the reason for an expression because of the malformed namespace prefixes.

Reply:
I agree with @jan-cerny . We should clean up our XMLs instead of adding these workarounds IMO. We will also need to add a root element to avoid forests of XML elements, forests are forbidden in well formed XML.

Reply:
Ahhhha, thanks for explanation, tbh I haven't tried parsing that.